tests/simpleKVBC/TesterReplica/strategy/MangledPreProcessResultMsgStrategy.hpp

```diff
@@ -0,0 +1,30 @@
+// Concord
+//
+// Copyright (c) 2018-2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the
+// LICENSE file.
+
+#pragma once
+
+#include "Logger.hpp"
+#include "TesterReplica/strategy/ByzantineStrategy.hpp"
+
+namespace concord::kvbc::strategy {
+class MangledPreProcessResultMsgStrategy : public IByzantineStrategy {
```

Comment:
As discussed over slack, please add documentation strings for both strategies (shuffle/mangle) in the header files.

Reply:
I have added documentation for both. I have updated the strategy in the changeMessage method and added a general comment on Shuffle header.